apps/apps.c

```diff
@@ -79,7 +79,7 @@ int chopup_args(ARGS *arg, char *buf)
 
     for (p = buf;;) {
         /* Skip whitespace. */
-        while (*p && isspace(_UC(*p)))
+        while (*p && ossl_isspace(_UC(*p)))
```

Comment:
Yes, there are cases where the input is from the local environment, and where input is from a standards-defined input (e.g. certificate). These need to be handled differently.